client/configuration/webpack/webpack.config.prod.js

```diff
@@ -108,14 +94,28 @@ module.exports = {
         minifyURLs: true
       }
     }),
-    new HtmlWebpackInlineSourcePlugin(),
-    new MiniCssExtractPlugin({
-      filename: "static/css/[name].[contenthash:8].css"
+    new CleanWebpackPlugin({
+      verbose: true,
+      protectWebpackAssets: false,
+      cleanAfterEveryBuildPatterns: ["main.js", "main.css"]
+    }),
+    new FaviconsWebpackPlugin({
+      logo: "./favicon.png",
+      prefix: "static/assets/",
+      favicons: {
+        icons: {
+          android: false,
+          appleIcon: false,
+          appleStartup: false,
+          coast: false,
+          firefox: false,
+          windows: false,
+          yandex: false
+        }
+      }
     }),
-    new SWPrecacheWebpackPlugin({
-      cacheId: "cellxgene",
-      filename: "service-worker.js"
-    })
+    new HtmlWebpackInlineSourcePlugin(HtmlWebpackPlugin),
```

Comment:
Minor note here. The repo for the `HtmlWebpackInlineSourcePlugin` says that it is [no longer maintained](https://github.com/DustinJackson/html-webpack-inline-source-plugin/blob/master/README.md) and asks people to use [this Facebook plugin](https://github.com/facebook/create-react-app/blob/edc671eeea6b7d26ac3f1eb2050e50f75cf9ad5d/packages/react-dev-utils/InlineChunkHtmlPlugin.js#L10). Would it be worthwhile to upgrade this now?

Reply:
yeah, noticed that.  I'd rather defer as this is known to work fine with our code base.